sync/src/block_status.rs

```diff
@@ -7,6 +7,7 @@ bitflags! {
         const HEADER_VALID            =     0b0000_0000_0000_0001;
         const BLOCK_RECEIVED          =     Self::HEADER_VALID.bits | 0b0000_0000_0000_0010;
         const BLOCK_STORED            =     Self::HEADER_VALID.bits | Self::BLOCK_RECEIVED.bits | 0b0000_0000_0000_1000;
+        const BLOCK_VALID             =     Self::HEADER_VALID.bits | Self::BLOCK_RECEIVED.bits | Self::BLOCK_STORED.bits | 0b0000_0000_0001_0000;
```

Comment:
`(1 << 4)` instead of `0b0000_0000_0001_0000` would be more readable?

Reply:
Thanks! Updated as you suggested.

And I add another [commit](https://github.com/nervosnetwork/ckb/pull/1237/commits/b4a37519169a1b480bf527ccbf1fd0d9a05e7e4b), PTAL.